libkmod/libkmod-elf.c

```diff
@@ -682,43 +712,34 @@ static int kmod_elf_get_symbols_symtab(const struct kmod_elf *elf,
 	if (strings[i - 1] != '\0')
 		count++;
 
-	/* sizeof(struct kmod_modversion) * count + size + 1 */
+	/* sizeof(struct kmod_modversion) * count + 1 */
```

Comment:
And with previously mentioned change, this additional 1 can be skipped.

Reply:
Indeed, as does the `count++` couple of lines above. Thanks o/